doc-content/jbpm-docs/src/main/asciidoc/CoreEngine/service-migration-con.adoc

```diff
@@ -3,6 +3,8 @@
 
 The process instance migration service is a utility for migrating process instances from one deployment to another. Process or task variables are not affected by the migration. However, the new deployment can use a different process definition.
 
+When migrating a process, the process instance migration service also automatically migrates all its subprocesses, subprocesses of its subprocesses, and so on.
```

Comment:
not all services just the endpoint added. also if you try to migrate a subprocess when the parent has not been migrated it fails.